marketplace-server/src/main/resources/application.properties

```diff
@@ -28,4 +28,12 @@ spring.mail.properties.mail.smtp.auth=true
 spring.mail.properties.mail.smtp.starttls.enable=true
 spring.mail.properties.mail.smtp.ssl.trust = smtp.gmail.com
 
+default-admin.name=AdminName
+default-admin.surname=AdminSurname
+default-admin.email=admin@admin.com
+default-admin.password=nimda
 
+default-user.name=UserName
+default-user.surname=UserSurname
+default-user.email=user@user.com
+default-user.password=resu
```

Comment:
do we keep passwords in the repo? especially admin

Reply:
It's a temporary measure. And this is not the only part of sensitive data that needs to be removed from the repository. We were planning to replace all sensitive data with environmental variables a long time ago. We are hoping to deal with this issue as fast as posible